sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,712 @@
+                for (ChatThreadInfo thread : firstPage.getItems()) {
+                    //TODO: Take further action
+                }
+                retrieveNextThreadPages(firstPage.getPageId(), pageCollection);
+            }
+
+            @Override
+            public void onFailure(Throwable throwable, Response response) {
+                // TODO: Display error message.
+            }
+        });
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+
+void retrieveNextThreadPages(String nextPageId,
+                       AsyncPagedDataCollection<ChatThreadInfo, Page<ChatThreadInfo>> pageCollection) {
+    pageCollection.getPage(nextPageId, new Callback<Page<ChatThreadInfo>>() {
+        @Override
+        public void onSuccess(Page<ChatThreadInfo> nextPage, Response response) {
+            for (ChatThreadInfo thread : nextPage.getItems()) {
+                //TODO: Take further action
+            }
+            if (nextPage.getPageId() != null) {
+                retrieveNextThreadPages(nextPage.getPageId(), pageCollection);
+            }
+        }
+
+        @Override
+        public void onFailure(Throwable throwable, Response response) {
+            // TODO: Display error message.
+        }
+    });
+}
+```
+
+### Update a thread
+
+Use the `update` method to update a thread's properties.
+
+```java
+// The new topic for the thread.
+final String topic = "updated topic";
+// The model to pass to update method.
+UpdateChatThreadRequest thread = new UpdateChatThreadRequest()
+    .setTopic(topic);
+
+// The unique ID of the thread.
+final String threadId = "<thread_id>";
+client.updateChatThread(threadId, thread, new Callback<Void>() {
+    @Override
+    public void onSuccess(Void result, Response response) {
+        //TODO: Take further action
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+```
+
+### Delete a thread
+
+Use `deleteChatThread` method to delete a thread.
+
+
+```java
+// The unique ID of the thread.
+final String threadId = "<thread_id>";
+client.deleteChatThread(threadId, new Callback<Void>() {
+    @Override
+    public void onSuccess(Void result, Response response) {
+        //TODO: Take further action
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+```
+
+## Message Operations
+
+### Send a message
+
+Use the `send` method to send a message to a thread.
+```java
+// The chat message content, required.
+final String content = "Test message 1";
+//  The display name of the sender, if null i.e. not specified, an empty name will be set.
+final String senderDisplayName = "An important person";
+// The message priority level, such as 'normal' or 'high', 
+// if null i.e. not specified, 'normal' will be set.
```

Comment:
Are priority levels lowercase? I saw they were uppercase in the iOS README.

Reply:
done!